doc/effects.md

```diff
@@ -88,17 +113,20 @@ square.addEffect(ScaleEffect(
 
 ## RotateEffect
 
-Applied to `PositionComponent`s, this effect can be used to rotate the component, using an [animation curve](https://api.flutter.dev/flutter/animation/Curves-class.html).
+Applied to `PositionComponent`s, this effect can be used to rotate the component, using an
+[animation curve](https://api.flutter.dev/flutter/animation/Curves-class.html).
 
-The angle (`radians`) is in radians and the speed is in radians per second, so if you for example want to turn 180° in 2 seconds you set `radians: pi` and `speed: 0.25`.
+The angle (`radians`) is in radians and the speed is in radians per second, so if you for example
```

Comment:
Maybe rework this sentence to say something like:
```suggestion
The `radians` argument defines the angle in radians and the `speed` argument is how fast it will rotate in radians per second, so if you for example
```